p5/requirements.txt

```diff
@@ -1,4 +1,4 @@
-astroid==2.11.1
+astroid==2.12.12
```

Comment:
This file doesn't seem to match the same file in https://github.com/eecs485staff/p5-search-engine/pull/634/files I'm not sure which one is right.

Reply:
Oops, I must've copied it over from the wrong branch. Should be good now.